docs/slotfilling.rst

```diff
@@ -89,6 +89,10 @@ You need to define three methods:
         return ["cuisine", "num_people", "outdoor_seating",
                 "preferences", "feedback"]
 
+    def validate_cuisine(self, value, dispatcher, tracker, domain):
```

Comment:
Let's not put it here, since there is an example of validate below

Reply:
@Ghostvv  I think it might be a good idea to put this empty function here, so the user can see how the validate function is part of the form action and sees it in context with the rest of the code in addition to having a detailed function below.

Reply:
actually no, this is probably not necessary and rather more confusing